crnk-core/src/main/java/io/crnk/core/queryspec/DefaultQuerySpecDeserializer.java

```diff
@@ -175,6 +175,11 @@ protected void deserializeFilter(QuerySpec querySpec, Parameter parameter) {
 		Class<?> attributeType = getAttributeType(querySpec, parameter.attributePath);
 		Set<Object> typedValues = new HashSet<>();
 		for (String stringValue : parameter.values) {
+			if (!typeParser.supports(attributeType) && typeParser.isMismatchedTypeAcceptedAsString()) {
```

Comment:
I would recommend to swap these two because of execution order.

Reply:
can do, but the supports method is rather cheap too. from a semantic point of view, I found that its the more important check. but if you insist, I'm fine with swapping them

Reply:
@remmeier has to decide. I was just suggesting :)

Reply:
anyway, this is commit is more of a check if this is a route we can / should go down.